### PR TITLE
New HPC Software guide

### LINKED ISSUE #92: new HPC software page
Make sure to include obvious use of /software
Modules

### PR DESCRIPTION
**Pull request by ChristinaLK**
Closes #92 

Qs for Tim: 
* are we going to have a build node?
* is there going to be local scratch for building?
* final installations go...where?
* what is he expecting to build in the module stack?

### CODE REVIEW

**Comment by ChristinaLK** [comment]
no build node, install to /software, can do everything in /software but /scratch is available if needed.

**Comment by ChristinaLK** [comment]
gcc + openmpi (netcdf eventually but not this round)
licensed software (COMSOL)

File: hpc-software.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+			$ ./configure --prefix=/software/user
+			~~~
+		This is where you will want to set the installation location to be your 
+		`/software` directory. 
+	1. `make` - this step compiles and links the code, turning it from human-readable 
+		source code to compiled binary code. This is usually the most time consuming 
+		step of the installation process. 
+	1. `make install` - this step copies compiled files to the final installation location 
+		(usually specified in the `configure` step). 
+
```
Comment (tjslauson) [inline hpc-software.md:134]: Consider adding a short section here about cleaning up build directories.  Some things you might save from an Autoconf-iscated software install are the stdout, stderr, and the `config.log` written out during the `./configure` step.  The rest is usually never used again after `make install`.

File: hpc-software.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+## Optional: Creating Your Own Modules
+
+TBD
```
Comment (tjslauson) [inline hpc-software.md:137]: Placeholder - are you OK with it published?

File: hpc-software.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+For groups that would like to share software installations among group
+members, please contact us about getting a shared "group" directory. 
+
+If you are new to software installation, see the section below for 
+a more step-by-step description of the process. 
+
+## Step by Step Process
+
+1. **Download Source Code** - download the source code for your desired program to 
+	your `/software` directory.
```
Comment (tjslauson) [inline hpc-software.md:110]: Recommending the source archive goes in `/software` is good for if the user wants to keep the source code around (good idea), but I'm wondering if it'll also lead to users leaving build cruft around in `/software` when they untar the source archive in the same directory and don't clean it up later.  Not the end of the world if (when) it happens, but a consideration.  Somewhere around here could be a good place to mention local scratch space.

File: hpc-software.md (hunk 2 above)
Comment (ChristinaLK) [reply hpc-software.md:137]: I'll comment out for now.

**Pull request merged**